tests/stub/iteration/scripts/v4x4/tx_pull_list.script

```diff
@@ -6,7 +6,7 @@ A: HELLO {"{}": "*"}
 C: BEGIN {"{}": "*"}
 S: SUCCESS {}
 C: RUN "*" "*" "*"
-S: SUCCESS {"fields": ["n"]}
+S: SUCCESS {"fields": ["n"], "qid": 0}
```

Comment:
I guess all other drivers fail when `qid` is there, while `Go` fails without it 🤔

Reply:
Other drivers fail at subsequent `PULL` message, since they don't implement `OPT_IMPLICIT_DEFAULT_ARGUMENTS`